chromium_src/chrome/browser/safe_browsing/extension_telemetry/extension_telemetry_service_factory.cc

```diff
@@ -31,15 +31,9 @@ ExtensionTelemetryServiceFactory::ExtensionTelemetryServiceFactory()
                                  ProfileSelections::BuildNoProfilesSelected()) {
 }
 
-content::BrowserContext*
-ExtensionTelemetryServiceFactory::GetBrowserContextToUse(
-    content::BrowserContext* context) const {
-  return nullptr;
```

Comment:
`ProfileSelections::BuildNoProfilesSelected` prevents a service being constructor in any profile, so this is not needed.